addons/crm_livechat/models/chatbot_script_step.py

```diff
@@ -73,6 +73,7 @@ def _process_step_create_lead(self, discuss_channel):
         return self.env["crm.lead"].create(create_values)
 
     def _process_step_create_lead_and_forward(self, discuss_channel):
+        operator_partner = discuss_channel.livechat_operator_id
```

Comment:
We can actually just save it before the `_process_step_forward_operator` which is the step that should change it.